dspace-api/src/main/java/org/dspace/content/authority/MetadataAuthorityServiceImpl.java

```diff
@@ -237,4 +242,31 @@ public int getMinConfidence(MetadataField metadataField)
         }
         return copy;
     }
+    
+
+	private void autoRegisterAuthorityFromInputReader() {
+		try {
+			DCInputsReader dcInputsReader = new DCInputsReader();
+			for (DCInputSet dcinputSet : dcInputsReader.getAllInputs(Integer.MAX_VALUE, 0)) {
```

Comment:
Again, I would move this logic to either `DCInputsReader` or `DCInputAuthority` as those classes are already focused on submission form config.

Reply:
At moment I prefer leave this method here. This method use the internal maps to register the authority plugin.